k8s/nginx-http/container/nginx.conf.template

```diff
@@ -45,6 +45,12 @@ http {
 
   keepalive_timeout 60s;
 
+  # Do not expose nginx data/version number in error response and header
+  server_tokens off;
+
+  # To prevent cross-site scripting
+  add_header X-XSS-Protection "1; mode=block";
```

Comment:
I am skeptical that this setting would break usage of JavaScript driver, where other websites can call `test.ipdb.io` API.

Reply:
I have tested the config with Python driver and direct REST calls and everything seems to be in place.